FW/Presentations/Lab_Guides/_C_01_Build_Setup_Download_EDK_II_Win_LabGuide.md

```diff
@@ -0,0 +1,350 @@
+
+Setup the Simics Simulator
+
+[Simics® Simulator Installation and Get Started](https://www.intel.com/content/www/us/en/developer/articles/guide/simics-simulator-installation.html)
+
+Open the "Intel Simics Package Manager"
+
+1. Using the Start menu in Windows, Left Click on "Windows Key" Lower Left
+2. Scroll down from the scroll bar on the right until "Intel Simics Package Manager"
+
+Here is a snapshot of Intel Simics Package Manager with "My Simics Project 1" created (see powerpoint for picture)
+
+
+---
+## Slide 11  Download EDK II Open Source Repos
+
+Use Git Bash to download EDK II
+
+
+---
+## Slide 12  Create Workspace Directory
+
+Open a Windows Command Prompt for VS
+Make new directory for Workspace:
+
+```
+$ cd /
+$ Mkdir FW
+$ cd FW
+$ Mkdir edk2-ws
+$ cd edk2-ws
+```
+
+---
+## Slide 13 Download the EDK II Source Code
+### Download the open source EDK II from GitHub
+
+Note:  if behind a firewall, set PROXYS FIRST (example shows for Intel Corp. -- Maybe different for your Corp. or GEO)
+```
+$ git config --global https.proxy proxy-dmz.intel.com:912
+$ git config --global http.proxy proxy-dmz.intel.com:911
+```
+
+From the command prompt use "git clone" to download following repos
+\#Edk2 - main core code
+```
+$ git clone -b Edk2Lab_22Q3 https://github.com/tianocore-training/edk2.git
+```
+\#EDK II "C" Library Repo
+```
+$ git clone https://github.com/tianocore/edk2-libc.git
+```
+\#EDK II Platforms Repo
+```
+$ git clone https:://github.com/tianocore/edk2-platforms.git
+```
+\#EDK II Non-OSI (Stand alone Binaries)
+```
+$ git clone https:://github.com/tianocore/edk2-non-osi.git
+```
+\#Intel FSP
+```
+$ git clone https://github.com/IntelFsp/FSP.git
+```
+
+---
+## Slide 14 Download the EDK II Source Code
+### Download the open source EDK II from GitHub
+
+Download the Submodules and Checkout the Lab Branch
+```
+$ C:\fw\edk2-ws> cd edk2
+$ C:\fw\edk2-ws\edk2> git submodule update --init
+$ C:\fw\edk2-ws> cd ..
+```
+
+Download Checkout the Sha tag for edk2-platforms repo
+```
+$ C:\fw\edk2-ws> cd edk2-platforms
+$ C:\fw\edk2-ws> git reset --hard c546cc01f1517b42470f3ae44d67efcb8ee257fc
+```
+
+(rest to this commit since this is used with all the labs)
+
+---
+## Slide 15  Download Lab_Material_FW - getting the Source 
+Download the Lab_Material_FW.zip from: [github.com Lab_Matrial_FW.zip](https://github.com/tianocore-training/Lab_Material_FW/archive/refs/heads/main.zip)
+OR
+Use `git clone` to download the Lab_Material_FW
+```bash
+C:\> git clone https://github.com/tianocore-training/Lab_Material_FW.git
+```
+Directory Lab_Material_FW will be created
+
+```
+   FW 
+    - Documentation 
+   - DriverWizard 
+   - edk2-ws      
+   - edk2Linux 
+   - LabSampleCode 
+   - Nasm
+```
+
+---
+## Slide 16 Build EDK II
+### Extract the Lab Material
+
+Extract the Downloaded Lab_Material_FW-main.zip to C:\
+
+
+Download the Lab_Material_FW.zip from 
```

Comment:
Adde this link: https://github.com/tianocore-training/Lab_Material_FW/archive/refs/heads/main.zip